pkg/k8scontext/types.go

```diff
@@ -21,6 +21,7 @@ type InformerCollection struct {
 	Namespace                    cache.SharedIndexInformer
 	AzureIngressManagedLocation  cache.SharedInformer
 	AzureIngressProhibitedTarget cache.SharedInformer
+	AzureAppGwBackendPool        cache.SharedInformer
```

Comment:
AzureApplicationGatewayBackendPool in full ?

Reply:
updated